utils/ansible_playbook_to_role.py

```diff
@@ -229,6 +234,19 @@ def _raw_playbook(self):
         with io.open(self._local_playbook_filename, 'r', encoding="utf-8") as f:
             return f.read()
 
+    @property
+    @memoize
+    def platform_version(self):
+        prod = self.name.split("_")
+        platform = prod[0]
+        # Check to see if this is RHEL product
+        if platform in PRODUCT_WHITELIST:
+            # For RHEL, we can get what version
+            if 'rhel' in platform:
+                return platform[len(platform)-1]
+            return "7\n    - 8"
```

Comment:
I think you don't plan on updating RHEV4 roles for the moment, but `PRODUCT_WHITELIST` contains `rhv4`, and this may lead to platform version 7 and 8 for RHEV.
Better disarm this trap now, :)

Reply:
I agree